inference-engine/src/inference_engine/src/ie_network_reader.cpp

```diff
@@ -282,6 +270,19 @@ CNNNetwork details::ReadNetwork(const std::string& model,
             return reader->read(modelStream, exts);
         }
     }
+    // Try to load with FrontEndManager
+    // NOTE: weights argument is ignored
+    static ngraph::frontend::FrontEndManager manager;
```

Comment:
Can we use one instance of `FrontEndManager` for core object?
Now you have 2 static instances